static/src/stylesheets/module/_adslot.scss

```diff
@@ -239,7 +239,7 @@
     }
 }
 
-.ad-slot--container-inline:not(.ad-slot--fluid) {
+.ad-slot--container-inline:not(.ad-slot--fluid):not(.ad-slot--gc) {
```

Comment:
Ideally this change should not be part of this PR, since the two changes are not related. 😄 

This is the fix for the GoogleCard ad-slot that we also wanted to get out today, so no worries!